Sources/StreamVideoSwiftUI/Livestreaming/LivestreamPlayer.swift

```diff
@@ -81,94 +81,15 @@ public struct LivestreamPlayer<Factory: ViewFactory>: View {
     public var body: some View {
         ZStack {
             if viewModel.errorShown {
-                Text(L10n.Call.Livestream.error)
+                errorView
             } else if viewModel.loading {
-                ProgressView()
+                loadingView
             } else if state.backstage {
-                Text(L10n.Call.Livestream.notStarted)
+                notStartedView
             } else {
-                ZStack {
```

Comment:
No changes here, just moving things around, right?

Reply:
that's right